shared/constants/field/myinfo/index.ts

```diff
@@ -49,7 +49,7 @@ export const types: MyInfoFieldBlock[] = [
   },
   {
     name: MyInfoAttribute.DateOfBirth,
-    value: 'Date of birth',
+    value: 'Date of Birth',
```

Comment:
This is a shared file. Will this affect anything on the server? Is this truly backwards compatible?

Reply:
yes and no :sob_smile: